.github/workflows/fetch-news.yml

```
@@ -0,0 +1,33 @@
+name: Fetch Discourse News
+
+on:
+  workflow_dispatch:    
+  schedule:
+    - cron: "0 6 * * *"  
+
+jobs:
+  update-news:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
+      - name: Install dependencies
+        run: npm ci || npm install
+
+      - name: Fetch latest news from Discourse
+        run: node tools/fetch-news.js
+
+      - name: Commit and push updated news.json
+        run: |
+          git config user.name "GitHub Action"
+          git config user.email "actions@github.com"
```

For consistency with the `update-submodules.yml`:


```suggestion
          git config --local user.name "precice-bot"
          git config --local user.email "info@precice.org"
```